.github/workflows/docs.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
```

Comment:
Let's please propagate this change also to other projects which are doing it the same way, in order to reduce overall resource consumption.